website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -21,6 +21,13 @@ After Consul 1.17.0, only timestamps will be included in rotated log file names.
 
 ## Consul 1.16.x
 
+#### Vault Enterprise as CA
```

Comment:
I think we should add to KNOWN ISSUES in the [changelog](https://github.com/hashicorp/consul/blob/main/CHANGELOG.md)

The changelog content can link to the applicable upgrade guidance section in our docs (here) once published.

The changelog PR can be separate (since it won't be backported in any way).